src/App.js

```diff
@@ -1,5 +1,7 @@
 import 'bootstrap/dist/css/bootstrap.min.css';
 import 'bootstrap/dist/js/bootstrap.bundle.min';
+import './App.css';
```

Comment:
Is any of the CSS in App.css actually being used? If not, get rid of it.

Reply:
App.css has been removed